MBBSEmu.Tests/ExportedModules/Majorbbs/alcdup_Tests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using MBBSEmu.Memory;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class alcdup_Tests : ExportedModuleTestBase
+    {
+        private const int ALCDUP_ORDINAL = 64;
+
+        [Theory]
+        [InlineData("TEST1\0", "TEST1\0")]
+        [InlineData("TEST1\0TEST2\0", "TEST1\0")]
+        [InlineData("TEST TEST test\0", "TEST TEST test\0")]
+        [InlineData("\0", "\0")]
+        [InlineData("", "\0")]
+        [InlineData(null, "\0")]
```

Comment:
these two tests are the same due to line 29 `IsNullOrEmpty`

You probably want to change that line to `if (string != null)`